source/triggers/database-triggers.txt

```diff
@@ -287,6 +287,35 @@ following operation types:
    * - **Replace**
      - Represents a new document that replaced a document in the collection.
 
+   * - **Create_Collection**
+     - Represents the creation of a new collection.
+   
+   * - **Modify_Collection**
+     - Represents the modification collection.
+
+   * - **Rename_Collection**
+     - Represents collection being renamed.
+
+   * - **Drop_Collection**
+     - Represents a collection being dropped.
+
+   * - **Shard_Collection**
+     - Represents a collection getting sharded.
+
+   * - **Reshard_Collection**
+     - Represents a change to a collection's sharding.
+
+   * - **Refine_Collection_Shard_Key**
```

Comment:
Not sure if this is possible, to update the spacing on this table as the `y` in this line is causing a new line.